smith

```diff
@@ -43,6 +43,12 @@ search "PHP Version"
 spit "Public ssh keys"
 search "ssh-rsa"
 
+spit "Phppgadmin instance"
+search "phppgadmin:"
+
+spit "etc/passwd"
```

Comment:
It might be better to write this as `/etc/passwd`.

Reply:
Not sure if dir traversal in the wordlist will help meg. If not those can be stripped from the wordlist.